debian/control

```diff
@@ -18,3 +18,27 @@ Suggests: python-oq-libs-extra (>= 1.5.0~)
 Description: computes seismic hazard and physical risk
  based on the hazard and risk libraries developed by the GEM foundation
  (http://www.globalquakemodel.org/)
+
+Package: python-oq-engine-master
+Architecture: all
+Depends: ${shlibs:Depends}, ${misc:Depends}, python-oq-libs (>=1.5.0~), rabbitmq-server (>=2.8.0-2~gem02), supervisor (>=3.0b2-1~), python-oq-engine (= ${binary:Version})
+Replaces: python-oq-risklib, python-oq-hazardlib
+# See https://www.debian.org/doc/debian-policy/ch-relationships.html#s-breaks
+Breaks: python-oq-risklib, python-oq-hazardlib
+Recommends: ${dist:Recommends}
+Suggests: python-oq-libs-extra (>= 1.5.0~)
+Description: computes seismic hazard and physical risk
+ based on the hazard and risk libraries developed by the GEM foundation
+ (http://www.globalquakemodel.org/)
+
+Package: python-oq-engine-worker
+Architecture: all
+Depends: ${shlibs:Depends}, ${misc:Depends}, python-oq-libs (>=1.5.0~), rabbitmq-server (>=2.8.0-2~gem02), supervisor (>=3.0b2-1~), python-oq-engine (= ${binary:Version})
```

Comment:
`python-oq-engine-worker` must not depend on RabbitMQ. I would probably also remove `python-oq-libs` since `master` depends on `engine` which depends already on `libs`. It's ok here to keep `supervisor` (we include an init in this pkg).